internal/bits/order_test.go

```diff
@@ -86,7 +86,7 @@ func isUndefined(ordering int) bool {
 }
 
 func isSorted(set sort.Interface) bool {
```

Comment:
How about renaming this to `isOrdered`? I feel like having it named `isSorted` but having a different behavior than `stort.IsSorted` is a bit confusing.

Reply:
👍 c229acb